topcoffea/cfg/2018_DEPRECATED_FORMAT.cfg

```diff
@@ -1,90 +0,0 @@
-path: /pool/phedex/userstorage/juanr/multilep21Feb2020/
-#options: 
-xsec : cfg/xsec.cfg
-year : 2018
-verbose: 1
-
-# Sample name : name of the files
-# Sample name --> The one in xsec.cfg and the output name
-
-### ttbar
-TTTo2L2Nu : TTTo2L2Nu_TuneCP5 
-#TTToSemiLeptonic : TTToSemiLeptonic_TuneCP5
-TTG  : TTGJets_TuneCP5_amcatnloFXFX_madspin
-
-# DY 
-DYJetsToLL_M_10to50_MLM : DYJetsToLL_M_10to50_TuneCP5_MLM
-DYJetsToLL_M_50_a : DYJetsToLL_M_50_TuneCP5_amcatnloFXFX, DYJetsToLL_M_50_TuneCP5_amcatnloFXFX_ext2
-
-### tW 
-tW_noFullHad : tW_5f_noFullHad_TuneCP5_ext1
-tbarW_noFullHad : tbarW_5f_noFullHad_TuneCP5_ext
-
-### WJets
-WJetsToLNu_MLM : WJetsToLNu_TuneCP5_MLM
-
-### WW
-WWTo2L2Nu : WWTo2L2Nu_NNPDF31_TuneCP5
-
-### WZ 
-# WZ
-WZTo2L2Q : WZTo2L2Q_amcatnloFXFX_madspin
-WZTo3LNu : WZTo3LNu_TuneCP5_ext1
-
-### ZZ
-ZZTo2L2Nu : ZZTo2L2Nu_TuneCP5_ext1
-ZZTo2L2Q : ZZTo2L2Q_amcatnloFXFX_madspin
-ZZTo4L :  ZZTo4L_TuneCP5_ext1, ZZTo4L_TuneCP5_ext2
-
-# ttW
-TTWJetsToLNu : TTWJetsToLNu_TuneCP5_amcatnloFXFX_madspin_ext1
-
-# ttZ
-TTZToLL_M_1to10 : TTZToLL_M_1to10_TuneCP5_amcatnlo
-TTZToLLNuNu_M_10_a: TTZToLLNuNu_M_10_TuneCP5_amcatnlo_ext1
-
-### Triboson
-WWW : WWW_4F_TuneCP5_amcatnlo_ext1
-WZG : WZG_TuneCP5_amcatnlo
-WWZ : WWZ_TuneCP5_amcatnlo_ext1
-WZZ : WZZ_TuneCP5_amcatnlo_ext1
-ZZZ : ZZZ_TuneCP5_amcatnlo_ext1
-#ZZG : ZZGJetsTo4L2Nu_4f_TuneCP5_amcatnloFXFX
-
-# tt raros
-tttt : TTTT_TuneCP5_amcatnlo_ext1
-ttWW : TTWW_TuneCP5_madgraph_ext1, TTWW_TuneCP5_madgraph_ext2
-ttWZ : TTWZ_TuneCP5_madgraph_ext1
-ttZH : TTZH_TuneCP5_madgraph_ext1
-ttZZ : TTZZ_madgraph_ext1
-ttHH : TTHH_TuneCP5_madgraph_ext1
-ttZZ : TTZZ_TuneCP5_madgraph_ext1
-#ttWH : TTWH_madgraph
-#TTTW : TTTW_madgraph
-#TTTJ : TTTJ, TTTJ_madgraph_ext1
-#ttGG : TTGG_0Jets_madspin
-#TTTW : TTTW_TuneCP5_madgraph_ext1
-
-tHq : THQ_4f_Hincl_madgraph
-ttHnobb : ttHToNonbb_M125_TuneCP5 #ttHJetToNonbb_M125_TuneCP5_amcatnloFXFX_madspin
-tZq : tZq_ll_4f_ckm_NLO_TuneCP5_madgraph_ext1
-
-#THW_5f_Hincl_madgraph
-#TTGG_0Jets_TuneCP5_amcatnlo_madspin
-#TTTJ_TuneCP5_madgraph_ext1
-#WWTo2L2Nu_DoubleScattering
-
-### Data
-# EGamma
-EGamma_2018 : EGamma_Run2018A_Nano25Oct2019_v1, EGamma_Run2018B_Nano25Oct2019_v1, EGamma_Run2018C_Nano25Oct2019_v1, EGamma_Run2018D_Nano25Oct2019_v1 : 10
-
-#MuonEG
-MuonEG_2018 : MuonEG_Run2018A_Nano25Oct2019_v1, MuonEG_Run2018B_Nano25Oct2019_v1, MuonEG_Run2018C_Nano25Oct2019_v1, MuonEG_Run2018D_Nano25Oct2019_ver2_v1 : 10
-
-#DoubleMuon
-DoubleMuon_2018 : DoubleMuon_Run2018A_Nano25Oct2019_v1, DoubleMuon_Run2018B_Nano25Oct2019_v1, DoubleMuon_Run2018C_Nano25Oct2019_v1, DoubleMuon_Run2018D_Nano25Oct2019_ver2_v1 : 10
-
-# Single Muon
-SingleMuon_2018 : SingleMuon_Run2018A_Nano25Oct2019_v1, SingleMuon_Run2018B_Nano25Oct2019_v1, SingleMuon_Run2018C_Nano25Oct2019_v1, SingleMuon_Run2018D_Nano25Oct2019_v1 : 10
-
-
```

Comment:
Is this file empty now, and should we delete it?

Reply:
Hmm, so I had actually attempted to removed this file. I do not see it in the `cfg` directory anymore, but I could be missing something. Where do you see the empty file?

Reply:
Ah, my mistake! I was thinking GitHub would mark it as removed, instead of showing all the lines as deleted. I do not see this in the `cfg` folder.